torch/_dynamo/variables/builtin.py

```diff
@@ -1026,6 +1025,10 @@ def call_method(
             return tx.output.side_effects.track_object_new_from_user_defined_class(
                 args[0]
             )
+        if self.fn is dict and name == "__new__":
+            assert len(args) == 1
+            assert len(kwargs) == 0
+            return ConstDictVariable({}, dict, mutation_type=ValueMutationNew())
```

Comment:
Should `dict` here be something like `args[0].as_python_constant()`?

Reply:
Here, the `self.fn` is dict (L1028). So, this is fine.

Reply:
Ah I misread the polyfill impl.